api/net/Areas/Admin/Controllers/ReportController.cs

```diff
@@ -193,7 +193,9 @@ public async Task<IActionResult> SendToAsync(int id, string to)
         {
             RequestorId = user.Id,
             To = to,
-            UpdateCache = true,
+            // no longer utilized, the caching mechanism
+            // determines whether to update or not
+            // UpdateCache = true,
```

Comment:
Do we need this to perhaps force an update under some condition?

Reply:
I was going to remove it, but noticed we use the property in some other cases via the message broker.  It does smell a bit funny to me...

Arguably we should switch to some sort of better solution that uses distributed caching rather than what is effectively a per service instance in memory cache. That of course is a bigger deal entirely.